model/sshable.rb

```diff
@@ -98,6 +98,11 @@ def cmd(cmd, stdin: nil, log: true)
     stdout_str
   end
 
+  def cmd_multiline(cmd, stdin: nil, log: true)
```

Comment:
Maybe use different name for the variable, so that it wouldn't get mixed up with the function. Maybe `command`?

Reply:
I've kept the same signature with `def cmd(cmd, stdin: nil, log: true)`. However, if you feel strongly about it, we can consider changing both

Reply:
I removed it